nx-X11/programs/Xserver/hw/nxagent/compext/Compext.h

```diff
@@ -827,19 +827,31 @@ extern int NXCollectProperty(
     Display*            /* display */,
     unsigned int        /* resource */,
     Window              /* window */,
```

Comment:
Don't we need an XlibWindow here (if defined), either?

Reply:
No! That's the whole trick here. In Clipboard.c we know about XlibWindow, in compext the very same datatype is called Window. Therefore this is correct.

However, there are more places in hw/nxagent/*c that need to be fixed reagarding XlibWindow. But this is not a quick task because this is like opening a big can of worms... I have started that and failed in the first approach. Generally spoken we need to adapt ALL datatypes that have an Xlib* datatype in Agent.h I am unsure (and I have not yet checked thoroughly ) if there are more datatypes that need to be added to Agent.h in and Xlib* variant.